### PR TITLE
Updates to README.md

### PR DESCRIPTION
**Pull request by yngki**
README.md updates for Nuvoton/Demo

Description
-----------
Updates to the README files to include Nuvoton.
Updates to the demo README for improved instructions. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by yngki** [comment]
can somebody please help me squash the commits :)?

**Comment by qiutongs** [comment]
> can somebody please help me squash the commits :)?

You want squash them when you submit by clicking "squash and merge"

File: README.md [hunk 1]
```diff
@@ -46,8 +46,14 @@ The following MCU boards are supported for Amazon FreeRTOS:
 13. **Marvell MW322** - [Marvell MW322 AWS IoT Starter Kit](https://www.marvell.com/microcontrollers/aws-iot-starter-kit/)
     * [Getting Started Guide](https://docs.aws.amazon.com/freertos/latest/userguide/getting_started_marvell322.html)
 14. **Nordic nRF52840 DK** - [nRF52840 DK Development kit](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK/)
-    * [Getting Started Guide](https://docs.aws.amazon.com/freertos/latest/userguide/getting_started_nordic.html)
-15. **Windows Simulator** - To evaluate Amazon FreeRTOS without using MCU-based hardware, you can use the Windows Simulator.
+    * [Getting Started Guide](https://docs.aws.amazon.com/freertos/latest/userguide/getting_started_nordic.html)  
+15. **Nuvoton** - [NuMaker-IoT-M487](https://direct.nuvoton.com/en/numaker-iot-m487)
+    * [Getting Started Guide](https://docs.aws.amazon.com/freertos/)
```
Comment (qiutongs) [inline README.md]: Is this link correct?

**Comment by sarenameas** [comment]
Should this go in release-candidate?

**Comment by yngki** [comment]
no longer needed

**Pull request closed without merging**